README.md

```diff
@@ -1,11 +1,43 @@
 # puppet-pulpcore
-Puppet module for setting up Pulp 3 as part of Katello installation
 
-# Pulpcore 3.2
+Puppet module to set up Pulp 3, also known as Pulpcore and its various plugins. The goal of this module is to provide a Puppet alternative to [pulp_installer](https://github.com/pulp/pulp_installer). The primary goal is to allow consumption within [foreman-installer](https://github.com/theforeman/foreman-installer), but should not be limited to that use case.
 
-We are adding a few new settings to make the installer compatible with pulpcore 3.2. These settings are not available in releases prior to 3.2 and should not be used in earlier versions.
+## Compatibility
 
-**ALLOWED_IMPORT_PATHS** : This setting whitelists paths that can be used for repository sync with file protocol. Katello uses the path /var/lib/pulp/sync_imports/ to run tests. For more information on this, see [https://docs.pulpproject.org/settings.html#allowed-import-paths](https://docs.pulpproject.org/settings.html#allowed-import-paths).
+The module requires at least Pulpcore 3.6.
```

Comment:
@jlsherrill I recall https://github.com/theforeman/puppet-foreman_proxy_content/commit/bd6654b038c6dc41518fa59469808a4a28a88a73 (which is sort of included in this PR) only worked with Pulpcore 3.6 so this would math it. Can you confirm this?

Going forward I would like this module to complete define the Pulp compatibility and puppet-foreman_proxy_content should only decide the compatibility on consuming some puppet-pulpcore version.